server/src-lib/Hasura/GraphQL/Execute/Action.hs

```diff
@@ -131,142 +130,76 @@ $(J.deriveJSON (J.aesonDrop 4 J.snakeCase){J.omitNothingFields=True} ''ActionHan
 instance L.ToEngineLog ActionHandlerLog L.Hasura where
   toEngineLog ahl = (L.LevelInfo, L.ELTActionHandler, J.toJSON ahl)
 
-resolveActionMutation
-  :: ( HasVersion
-     , MonadReusability m
-     , MonadError QErr m
-     , MonadReader r m
-     , MonadIO m
-     , Has FieldMap r
-     , Has OrdByCtx r
-     , Has SQLGenCtx r
-     , Has HTTP.Manager r
-     , Has [HTTP.Header] r
-     , Has (L.Logger L.Hasura) r
-     , Tracing.MonadTrace m
-     , MonadIO tx
-     , MonadTx tx
-     , Tracing.MonadTrace tx
-     )
-  => Env.Environment
-  -> Field
-  -> ActionMutationExecutionContext
-  -> UserInfo
-  -> m (tx EncJSON, HTTP.ResponseHeaders)
-resolveActionMutation env field executionContext userInfo =
-  case executionContext of
-    ActionMutationSyncWebhook executionContextSync ->
-      resolveActionMutationSync env field executionContextSync userInfo
-    ActionMutationAsync ->
-      (,[]) <$> resolveActionMutationAsync field userInfo
+
+-- resolveActionMutation
+--   :: ( HasVersion
+--      , MonadError QErr m
+--      , MonadIO m
+--      , MonadReader r m
+--      , Has HTTP.Manager r
+--      , Has [HTTP.Header] r
+--      )
+--   => Env.Environment
+--   => Field
+--   -> ActionMutationExecutionContext
+--   -> UserVars
+--   -> m (RespTx, HTTP.ResponseHeaders)
+-- resolveActionMutation env field executionContext sessionVariables =
+--   case executionContext of
+--     ActionMutationSyncWebhook executionContextSync ->
+--       resolveActionMutationSync env field executionContextSync sessionVariables
+--     ActionMutationAsync ->
+--       (,[]) <$> resolveActionMutationAsync field sessionVariables
+
```

Comment:
Remove the commented code?

Reply:
@rakeshkky this function seems used on master; is this okay?

Reply:
@codingkarthik Feel free to remove the commented code.